main.tf

```diff
@@ -1,8 +1,8 @@
 terraform {
   required_providers {
-    gsuite = {
-      source  = "DeviaVir/gsuite"
-      version = "0.1.58"
+    googleworkspace = {
+      source  = "hashicorp/googleworkspace"
+      version = "0.7.0"
```

Comment:
@tyriis @jazzlyn  it is not practical do define version somewhere outside in variables? So in future you don't need to change it in all files, but just in one place.